tests/Agent/IntegrationTests/SharedApplications/Common/MFALatestPackages/MFALatestPackages.csproj

```diff
@@ -53,8 +53,9 @@
     <PackageReference Include="NEST" Version="7.17.5" Condition="'$(TargetFramework)' == 'net481'" />
     <PackageReference Include="NEST" Version="7.17.5" Condition="'$(TargetFramework)' == 'net8.0'" />
     
-    <PackageReference Include="NLog" Version="5.3.2" Condition="'$(TargetFramework)' == 'net481'" />
-    
+    <PackageReference Include="NLog" Version="5.3.3" Condition="'$(TargetFramework)' == 'net481'" />
+    <PackageReference Include="NLog" Version="5.3.3" Condition="'$(TargetFramework)' == 'net8.0'" />
```

Comment:
I feel like there was a reason I didn't include the net8.0 target previously. But I guess if the tests pass, we're all good.

Reply:
Yeah, I wondered if there was a reason, but everything builds and passes. 🤷‍♂️